src/sql/src/statements/create.rs

```diff
@@ -110,7 +111,8 @@ impl CreateTable {
         if self.options.is_empty() {
             "".to_string()
         } else {
-            let options = format_list_indent!(self.options);
+            let options: Vec<&SqlOption> = self.options.iter().sorted().collect();
```

Comment:
`test_show_create_external_table_sql` sometimes fails due to `self.options` not being sorted. So, I appended this.

Reply:
This looks great. Stable ordering is also helpful for testing